src/components/Range.js

```diff
@@ -85,11 +76,20 @@ function Range({ id, min, max, value, step, onChange }) {
         min={min}
         max={max}
         onChange={onChange}
-        step={step}
         value={value}
       />
     </Container>
   );
 }
+Range.defaultProps = {
+  step: 1,
+};
+Range.propTypes = {
+  id: PropTypes.string,
+  min: PropTypes.number,
+  max: PropTypes.number,
+  onChange: PropTypes.func,
+  value: PropTypes.number,
```

Comment:
These props should be required. Don't forget to add `label` too. And `step` is also missing.